### PR TITLE
Creating UrlTree to determine if link is active.

### PR DESCRIPTION
**Pull request by PearseMcMurray**
https://jira.autonomy.com/browse/EL-2897

### CODE REVIEW

File: src/components/navigation/navigation-item/navigation-item.component.ts [hunk 1]
```diff
@@ -1,35 +1,21 @@
...
-    Output,
-    AfterViewInit,
-    AfterContentInit,
-    OnDestroy
-} from '@angular/core';
-import { Router, NavigationEnd, UrlTree } from '@angular/router';
+import { ActivatedRoute, NavigationEnd, Router, RouterState, UrlTree } from '@angular/router';
+import { AfterContentInit, AfterViewInit, Component, ContentChildren, ElementRef, Input, OnDestroy, Optional, QueryList, Renderer2, SkipSelf, HostBinding } from '@angular/core';
 import { filter } from 'rxjs/operators/filter';
+import { Observable } from 'rxjs/Observable';
```
Comment (ashley-hunter) [inline src/components/navigation/navigation-item/navigation-item.component.ts]: I don't think this is used so can probably be removed

File: src/components/navigation/navigation-item/navigation-item.component.ts [hunk 2]
```diff
@@ -85,14 +68,20 @@ export class NavigationItemComponent implements AfterViewInit, AfterContentInit,
         this._childrenChanges = this._children.changes.subscribe(() => this.setIndentWithoutArrow());
     }
 
-    ngOnDestroy () {
+    ngOnDestroy() {
```
Comment (ashley-hunter) [inline src/components/navigation/navigation-item/navigation-item.component.ts]: Add return type for consistency

File: src/components/navigation/navigation-item/navigation-item.component.ts (hunk 1 above)
Comment (PearseMcMurray) [reply src/components/navigation/navigation-item/navigation-item.component.ts]: Import removed.

File: src/components/navigation/navigation-item/navigation-item.component.ts (hunk 2 above)
Comment (PearseMcMurray) [reply src/components/navigation/navigation-item/navigation-item.component.ts]: Return type added.

**Pull request merged**